src/Equations/Fluid/HottailRateTermLowZ.cpp

```diff
@@ -0,0 +1,423 @@
+/**
+ * Implementation of equation term representing the runaway generation rate
+ * due to hottail when using an analytic distribution function
+ */
+
+#include "DREAM/Equations/Fluid/HottailRateTermLowZ.hpp"
+
+using namespace DREAM;
+
+/**
+ * Constructor.
+ */
+HottailRateTermLowZ::HottailRateTermLowZ(           
+    FVM::Grid *grid, AnalyticDistributionHottail *dist, FVM::UnknownQuantityHandler *unknowns,
+    IonHandler *ionHandler, CoulombLogarithm *lnL, RunawayFluid *runawayFluid, real_t sf
+) : HottailRateTerm(grid, dist, unknowns,sf), lnL(lnL), ionHandler(ionHandler), runawayFluid(runawayFluid),  
+    id_ncold(unknowns->GetUnknownID(OptionConstants::UQTY_N_COLD)),
+    id_Efield(unknowns->GetUnknownID(OptionConstants::UQTY_E_FIELD)), // Used for constant, not in jacobian.
+    id_tau(unknowns->GetUnknownID(OptionConstants::UQTY_TAU_COLL)),
+    id_Tcold(unknowns->GetUnknownID(OptionConstants::UQTY_T_COLD)), // Needed for derivatives in jacobian
+    id_johm(unknowns->GetUnknownID(OptionConstants::UQTY_J_OHM)), // Used for constant, not in jacobian. 
+    id_ni(unknowns->GetUnknownID(OptionConstants::UQTY_ION_SPECIES)) // Needed for derivatives in jacobian
+{
+    SetName("HottailRateTermLowZ");
+
+    //AddUnknownForJacobian(unknowns,id_Efield);
+    AddUnknownForJacobian(unknowns,id_ncold);
+    AddUnknownForJacobian(unknowns,id_tau);
+    AddUnknownForJacobian(unknowns,id_ni); //Also affects Zeff, now we neglect this contribution. 
+    AddUnknownForJacobian(unknowns,id_Tcold);
+    
+    this->w = gsl_integration_workspace_alloc(1000);
```

Comment:
It would probably be good if this constants (``1000``) could be defined in one place, since it's used in several places. You could e.g. add a ``const len_t nGslIntervals=1000`` to the class definition in the ``.hpp`` file.